components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/api/client/GradleEnterpriseApiClient.java

```diff
@@ -122,7 +122,7 @@ private Duration parseTimeout(String key) {
         }
     }
 
-    public BuildValidationData fetchBuildValidationData(NumberedBuildScan buildScan) {
+    public BuildScanData fetchBuildValidationData(NumberedBuildScan buildScan) {
```

Comment:
Rename method to `fetchBuildScanData`.

Reply:
Done in d8d9203f6898a5b752c888e77339cbed23813389.

Reply:
LGTM